examples/k8s/deployment.yaml

```diff
@@ -40,6 +40,29 @@ spec:
           image: registry.example.com/argoslower:latest
           args:
             - "--log-level=debug"
+            - "--enable-webhook-controller"
+            - "--supported-hooks=github=any,officeips=any,jira=any"
+#            - "--supported-hooks=github=any,officeips=officeips,jira=file"
+          env:
+            - name: "APP_IPFILE"
+              value: "/etc/argoslower/iplist.yaml"
+            - name: "APP_IPFILE_SOURCES"
+              value: "jira"
```

Comment:
I don't think having env var `APP_IPFILE_SOURCES` is useful. Currently there is just 1 source `jira`. But what if there were 2?
How to correlate the supported-hooks of file to the matching section?
```
    iplists:
      jira:
      - 192.168.0.0/16
      newsource:
      - 1.1.1.1/32
```

I think a simpler way is to get rid of `APP_IPFILE_SOURCES` and hardcode that any `supported-hooks` of `key=file` will look for section of `key`

Reply:
```
- "--supported-hooks=github=any,officeips=any,jira=file,otherthing=file"
```
```

Reply:
the issue is this [code](https://github.com/kanopy-platform/argoslower/blob/Feature_eventsource_controller/internal/cli/cli.go#L276-L277), it'll use `jira` and `otherthing` for both

Reply:
Got it. I clearly didn't grok how to use that file reader.